src/main/webapp/paginationScript.js

```diff
@@ -56,7 +56,19 @@ function displayCards(listAugment) {
 
     resultsContent.innerHTML = '';
     for (let i = currentFirstCardIndex; i < (currentFirstCardIndex + TOTAL_CARDS_TO_DISPLAY); i++) {
+      if (photoReferencesArray[i] != 'none') {
+        loadImage(listingsArray[i].children[0].children[0], photoReferencesArray[i]);
+      }
+
       resultsContent.appendChild(listingsArray[i]);
     }
   }
-}  
\ No newline at end of file
+}  
+
+function loadImage(listingImage, photoReference) {
```

Comment:
Have you thought about adding a placeholder image for the listings that don't have a photoreference?

Reply:
Yes, working on this just didn't get it added in this pr